modules/apps/portal-workflow/portal-workflow-instance-tracker-web/src/main/resources/META-INF/resources/js/util/util.js

```diff
@@ -22,6 +22,11 @@ import JoinNode from '../components/nodes/hexagon-nodes/JoinNode';
 import JoinXorNode from '../components/nodes/hexagon-nodes/JoinXorNode';
 import BorderStateNode from '../components/nodes/state-node/BorderStateNode';
 import StateNode from '../components/nodes/state-node/StateNode';
+import Edge from '../components/transitions/Edge';
+
+const edgeTypes = {
+	custom: Edge,
+};
```

Comment:
Hey @naraandrade, What do you think if we rename this to `transition` or something similar? Thx

Reply:
`custom` is too generic